### DISCUSSION TITLE
getting warning in the cluster operator: 'kafkas' with unstable version 'v1beta2'

### DISCUSSION THREAD

**Discussion by nazar200** [strimzi/strimzi-kafka-operator#8115] [Q&A]
I am getting WARNING in my operator log saying "client is using resource type 'kafkas' with unstable version 'v1beta2'

In all the examples I see v1beta2, Any issues if we are using this. Do we have any stable version?

**Comment by scholzj** [comment]
You can just ignore it. That is a warning you get for any resource which has some `alpha` or `beta` version`.

**Reply by nazar200** [reply]
Thanks @scholzj : Do we have any other version, which I can use to avoid this Warning

**Reply by scholzj** [reply]
No.
